source/sync/configure/sync-settings.txt

```diff
@@ -19,21 +19,6 @@ Available Settings
 ------------------
 
 .. _flexible-sync:
-
-Sync Type
-~~~~~~~~~
-
-Atlas Device Sync has two sync modes: Flexible Sync and the older
-Partition-Based Sync. We recommend using Flexible Sync. For information about
```

Comment:
I don't think we can delete this section entirely, but we should update it. When you go to enable Sync, the App Services UI still shows Partition-Based Sync, although it is grayed out. The UI says:

> Partition Based sync has been deprecated and is disallowed for new Sync configurations. Learn more about the differences between [Partition Based Sync and Flexible Sync](https://www.mongodb.com/docs/atlas/app-services/sync/configure/sync-settings/#sync-type)

And the link goes directly to this section in the docs.

I think we should leave this section, but update it to copy the language in the UI and say something like:

> Atlas Device Sync has two sync modes: Flexible Sync and the older Partition-Based Sync. Partition-Based Sync has been deprecated and is disallowed for new Sync configurations. If you have an existing app that uses Partition-Based Sync, you can migrate to Flexible Sync. For more information, refer to :ref:\`realm-sync-migrate-modes`.

Reply:
Changed! As the UI call out says to go to this page for more information on the differences, I added link to PBS page for more context as well if anyone is interested to learn more. Let me know what you think.

Reply:
Since Partition-Based Sync is deprecated, and is disallowed for new apps, I don't _think_ we need to link to it from here. This page is mainly for people who are setting up Sync for the first time, and I don't think we need to send them out to another page to read about a sync mode they can't even use. I think the wording in the UI is unfortunate for the same reason - if we're not allowing people to create new PBS apps, the differences between PBS and FS don't matter. FS is the only Sync they can use.